bindings/rust/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 
+extern crate core;
```

Comment:
Not sure if we need this. Seems to build okay without this.

Reply:
Fixed in fb41231